templates/ansible/stack-provisioner-hieradata.j2

```diff
@@ -116,6 +116,7 @@ aem_curator::config_publish::publish_timeout: {{ aem.client_timeout }}
 aem_curator::config_aem_tools::oak_run_version: '{{ library.oak_run_version }}'
 aem_curator::config_aem_tools::enable_compaction_remove_bak_files: {{ aem.enable_bak_files_cleanup }}
 aem_curator::config_aem_tools::compaction_remove_bak_files_older_than_in_days: '{{ aem.bak_files_cleanup_max_age_in_days }}'
+aem_curator::config_aem_tools::compaction_jvm_mem_opts: '{{ aem.compaction.jvm_mem_opts }}'
```

Comment:
According to the puppet-aem-curator manifest
https://github.com/shinesolutions/puppet-aem-curator/blob/master/manifests/config_aem_tools.pp#L13

this should be named ```aem_curator::config_aem_tools::aem_compaction_jvm_mem_opts```